### DISCUSSION TITLE
Added v0.12, v1.0 and v1.1 with easy install

### DISCUSSION THREAD

**Discussion by zcb-code** [deepmodeling/deepmd-kit#1581] [Ideas]
Is that possible to added  v0.12, v1.0 and v1.1 with easy conda install? Because some models have already trained with these versions and supplied in supporting information. And it is difficult to install for a beginner.

**Comment by njzjz** [comment]
The offline packages of old versions were archived at https://github.com/deepmd-kit-recipes/deepmd-kit-recipes/releases.
Newer versions can be found at https://github.com/deepmodeling/deepmd-kit/releases.

**Reply by zcb-code** [reply]
Thanks
